src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -842,21 +875,84 @@ public static int getSpecificID(String value) {
 		return id;
 	}
 
-	public static String getSampleFrequency(String value) {
-		/*
-		 * Some tracks show several values, e.g. "48000 / 48000 / 24000" for
-		 * HE-AAC. We store the first value only.
-		 */
-		if (value.indexOf('/') > -1) {
-			value = value.substring(0, value.indexOf('/'));
+	public static int[] parseSampleRate(String value) {
+		if (isBlank(value)) {
+			LOGGER.debug("MediaInfo returned a blank sample rate value, falling back to default");
+			return new int[0];
+		}
+
+		// examples of libmediainfo output (mediainfo --Full --Language=raw file):
+		// SamplingRate : 48000
+		// SamplingRate : 44100 / 22050
+		// SamplingRate : 48000 / 48000 / 24000
+
+		ArrayList<Integer> rates = new ArrayList<>();
+		Matcher intMatcher = intPattern.matcher(value);
+		while (intMatcher.find()) {
+			String matchResult = intMatcher.group();
+			try {
+				rates.add(0, Integer.valueOf(matchResult));
+			} catch (NumberFormatException ex) {
+				LOGGER.warn("NumberFormatException while parsing substring {} from value {}", matchResult, value);
+			}
 		}
 
-		if (value.contains("khz")) {
-			value = value.substring(0, value.indexOf("khz"));
+		if (rates.isEmpty() || rates.get(0).intValue() == 0) {
+			LOGGER.warn("Can't parse the sample rate from \"{}\"", value);
+		}
+		int[] result = new int[rates.size()];
+		for (int i = 0; i < rates.size(); i++) {
+			result[i] = rates.get(i).intValue();
 		}
+		return result;
+	}
 
-		value = value.trim();
-		return value;
+	public static int parseDelay(String value) {
+		if (isBlank(value)) {
+			LOGGER.debug("MediaInfo returned a blank delay value, falling back to default");
+			return 0;
+		}
+
+		// examples of libmediainfo output (mediainfo --Full --Language=raw file):
```

Comment:
@Sami32 Pardon my French, do you mean that the ```E``` in ```examples``` should be capitalized? I didn't write or change the text, I only moved it.